interview-questions.md

```diff
@@ -15,54 +15,55 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: Git is version control software - local
+  Github is a repository (web based) - allows distribution, sharing, etc
 
-  Researched answer:
+  Researched answer: Git is a locally installed tool used to manage source code. Github allows for storage, sharing of Git.
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: logical operators - And, Not, Or
 
-  Researched answer:
+  Researched answer:Logical operators return true or false - yes or no
```

Comment:
There are three operators that produce that Boolean value of true or false:
Look at Boolean values
https://github.com/learn-academy-2022-bravo/Syllabus/blob/main/javascript/intro.md

Logical (&&, ||, !)
Relational (>, <, <=, >=)
Equality (===, !==)